build.gradle

```diff
@@ -0,0 +1,40 @@
+plugins { 
```

Comment:
Please use a `build.gradle` that is similar to the existing ones. If each fixture starts writing its build.gradle from scratch they all differ. See https://github.com/test-editor/rest-fixture/blob/develop/build.gradle